README.md

```diff
@@ -1 +1,36 @@
-# Conference
+# Marketing Conference
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Created a global marketing conference website. It has Home Page and About Page and also added the user interaction link, mobile menu, Implement the section "Featured speakers" with dynamic HTML.
+In mobile version when 'MORE' button is dynamically used and followed the design.
+
+
```

Comment:
Kindly add a screenshot of your page in the README file in order for the README file to be more professional.